### ISSUE TITLE
Example does not work with onnxruntime 1.0

### ISSUE THREAD

**Issue by MaxNoe** [onnx/sklearn-onnx#303]
The example in the docs landing page throws this error with onnxruntime 1.0:

```
InvalidArgument                           Traceback (most recent call last)
<ipython-input-2-3cce4c8fc66e> in <module>
     22 input_name = sess.get_inputs()[0].name
     23 label_name = sess.get_outputs()[0].name
---> 24 pred_onx = sess.run([label_name], {input_name: X_test.astype(numpy.float32)})[0]

~/.local/anaconda3/lib/python3.7/site-packages/onnxruntime/capi/session.py in run(self, output_names, input_feed, run_options)
    134             output_names = [output.name for output in self._outputs_meta]
    135         try:
--> 136             return self._sess.run(output_names, input_feed, run_options)
    137         except C.EPFail as err:
    138             if self._enable_fallback:

InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Got invalid dimensions for input: float_input for the following indices
 index: 0 Got: 38 Expected: 1
 Please fix either the inputs or the model.
```

**Comment by prabhat00155** [comment]
Thanks for pointing this. Shape in the initial type needs to be fixed for 1.0. It should be:
initial_type = [('float_input', FloatTensorType([None, 4]))]

**Comment by xadupre** [comment]
The documentation has just been updated. I'll close the issue.
